spec/controllers/users_controller_spec.rb

```diff
@@ -18,6 +18,32 @@
         expect(response).to redirect_to sign_in_path
       end
     end
+
+    context "with valid input and invitation token" do 
+      it "makes the user follow the inviter" do 
+        user = Fabricate(:user)
```

Comment:
You could refactor `user = Fabricate(:user)` to `let(:user) { Fabricate(:user) }` you are using it in 3 below examples.